melodies_monet/driver.py

```diff
@@ -1756,7 +1759,8 @@ def plotting(self):
                             if filter_criteria and 'altitude' in filter_criteria:
                                 vmin_y2, vmax_y2 = filter_criteria['altitude']['value']
                             elif filter_criteria is None:
-                                if 'altitude' in pairdf.columns:
+                                #if 'altitude' in pairdf.columns: # pairdf is dataset object, don't have columns
+                                if 'altitude' in pairdf.variables: # for xarray that might be a coordinate variable
```

Comment:
```suggestion
                                if 'altitude' in pairdf:
```
@blychs said https://github.com/NOAA-CSL/MELODIES-MONET/pull/282#issuecomment-2394967837